native/calendar/entry.react.js

```diff
@@ -392,19 +391,13 @@ class InternalEntry extends React.Component<Props, State> {
             iosActiveOpacity={opacity}
           >
             <View>
-              <View style={heightStyle} />
-              <Hyperlink
-                linkDefault={true}
-                linkStyle={linkStyle}
-                style={this.props.styles.textContainer}
+              <View style={[heightStyle, this.props.styles.textContainer]} />
```

Comment:
Why did you add `this.props.styles.textContainer` to the style here?

Reply:
To make this look like before, without `this.props.styles.textContainer` it was bigger and looked like this:
<img width="329" alt="Zrzut ekranu 2020-07-14 o 14 02 15" src="https://user-images.githubusercontent.com/18313471/87423473-aa5d3900-c5da-11ea-80d4-72cb88056d2c.png">
instead of 
<img width="339" alt="Zrzut ekranu 2020-07-14 o 14 02 23" src="https://user-images.githubusercontent.com/18313471/87423477-ab8e6600-c5da-11ea-9a24-9b77b32cc543.png">

Reply:
That does seem like an issue, but I'm concerned that you are overriding `heightStyle` with `this.props.styles.textContainer` here. I'll look into it...

Reply:
So what you're doing here is basically making `heightStyle` useless. `textContainer` is `position: absolute`, which means this `View`'s `height` will only apply to itself.

It's important that the height of this `FlatList` item matches `heightStyle`. Otherwise our ability to calculate the position of elements in the `FlatList` could be broken.

I got it working this way:

```js
<View style={heightStyle} />
<View style={this.props.styles.textContainer}>
  <Text
    style={[this.props.styles.text, textStyle]}
    onLayout={this.onTextLayout}
  >
    <Markdown darkStyle={darkColor}>{rawText}</Markdown>
  </Text>
</View>
```

In the future, remember that it's not just about making things look correct. When you change a line of code, make sure you understand 100% why it was there before, and what it was doing.